examples/stars_html.php

```diff
@@ -0,0 +1,5 @@
+<?php
+
+require_once(__DIR__ . "/bootstrap.php");
+
+echo $rivio->stars_html('1492411012');
```

Comment:
ez is cacheből olvasson

ha nincs  cacheben ilyen file, akkor ne dobjon hibát (product_reviews_htmlre is)